plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/handler/ResultSetPropertyTester.java

```diff
@@ -49,6 +49,7 @@
     public static final String PROP_CAN_NAVIGATE_HISTORY = "canNavigateHistory";
     public static final String PROP_EDITABLE = "editable";
     public static final String PROP_CHANGED = "changed";
+    private static final String PROP_UPDATE_NOT_IN_PROGRESS = "updateNotInProgress";
```

Comment:
Generally, I wouldn't say I like the idea of providing properties that return negative/inverted values.

It would be more convenient to return whether the update is **in** progress or not.